evalml/automl/automl_search.py

```diff
@@ -385,6 +385,7 @@ def __init__(self,
         check_all_pipeline_names_unique(self.allowed_pipelines)
 
         run_ensembling = self.ensembling
+        text_in_ensembling = run_ensembling if infer_feature_types(X_train).select('natural_language').columns else False
```

Comment:
Ah right I forgot that under our current API, the automl algorithm doesn't have access to the data directly.

Is `run_ensembling` necessary to use here? Could you do this instead
```python
text_in_ensembling = len(infer_feature_types(X_train).select('natural_language').columns) > 0
```

Reply:
Much cleaner, thanks!